ch3-selectors/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Selectors</title>
+    <link rel="stylesheet" type="text/css" href="style.css" media="screen" />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <h1>Selectors</h1>
+      </header>
+      <main>
+        <div>
+          <label>atrr / attr= / attr~= / attr|= </label>
+          <ul>
+            <li>
+              <a href="" lang="en" rel="no so Exact">I have a similar value</a>
+            </li>
+            <li>
+              <a href="" lang="es" rel="similar but no Exact"
+                >I have a similar value</a
```

Comment:
Acá sobraron unos enters

Reply:
Ya vi el error, el plug in que uso para que me borre espacios se volvió loco hahaha pero ya está